react/react1/week1/react-week1/src/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App.js';
+import reportWebVitals from './reportWebVitals';
+
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+     <h3 className='TodoTitle'>TodoList</h3>
+     <> <App /> </>
```

Comment:
I would remove the fragments around App as it is not needed because your components are already wrapped in a parent and therefore we don't need the fragments.